components/zoho_desk/sources/updated-ticket/updated-ticket.mjs

```diff
@@ -0,0 +1,48 @@
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "zoho_desk-updated-ticket",
+  name: "New Updated Ticket",
+  description: "Emit new event when a ticket is updated. [See the docs here](https://desk.zoho.com/DeskAPIDocument#Tickets#Tickets_Listalltickets)",
+  type: "source",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    orgId: {
+      propDefinition: [
+        common.props.zohoDesk,
+        "orgId",
+      ],
+    },
+  },
+  methods: {
+    ...common.methods,
+    getResourceFn() {
+      return this.zohoDesk.searchTickets;
+    },
+    getResourceFnArgs() {
+      return {
+        headers: {
+          orgId: this.orgId,
+        },
+        params: {
+          sortBy: "modifiedTime", // relevance | modifiedTime | createdTime | customerResponseTime
+        },
+      };
+    },
+    resourceFilter(resource) {
+      const lastUpdatedAt = this.getLastUpdatedAt() || 0;
+      return Date.parse(resource.modifiedTime) > lastUpdatedAt;
+    },
+    generateMeta(resource) {
+      const ts = Date.parse(resource.modifiedTime);
+      return {
+        id: ts,
```

Comment:
Maybe include the id just in case there are 2 resources with the same modifiedTime?
```suggestion
        id: `${resource.id}${ts}`,
```